src/Views/SuccessView.vala

```diff
@@ -45,37 +46,33 @@ public class Sideload.SuccessView : AbstractView {
-                    /// TRANSLATORS: "System Settings" is related to the title of https://github.com/elementary/switchboard, "Applications" is related to the title of https://github.com/elementary/switchboard-plug-applications
-                    appstore_name, "settings://applications/permissions", _("System Settings"), _("Applications")
-                );
-            } else if (file is FlatpakBundleFile) {
-                secondary_label.label = _("Open it any time from the Applications Menu. Permissions can be changed in <a href='%s'>%s → %s…</a>").printf (
-                    /// TRANSLATORS: "System Settings" is related to the title of https://github.com/elementary/switchboard, "Applications" is related to the title of https://github.com/elementary/switchboard-plug-applications
-                    "settings://applications/permissions", _("System Settings"), _("Applications")
-                );
-            }
+            secondary_label_string = _("Open it any time from the Applications Menu.");
+
         } else if (view_type == SuccessType.ALREADY_INSTALLED) {
             if (app_name != null) {
                 primary_label.label = _("“%s” is already installed").printf (app_name);
             } else {
                 primary_label.label = _("This app is already installed");
             }
 
-            if (file is FlatpakRefFile) {
-                secondary_label.label = _("No changes were made. Visit %s for app information, updates, and to uninstall. Permissions can be changed in <a href='%s'>%s → %s…</a>").printf (
-                    /// TRANSLATORS: "System Settings" is related to the title of https://github.com/elementary/switchboard, "Applications" is related to the title of https://github.com/elementary/switchboard-plug-applications
-                    appstore_name, "settings://applications/permissions", _("System Settings"), _("Applications")
-                );
-            } else if (file is FlatpakBundleFile) {
-                secondary_label.label = _("No changes were made. Permissions can be changed in <a href='%s'>%s → %s…</a>").printf (
-                    /// TRANSLATORS: "System Settings" is related to the title of https://github.com/elementary/switchboard, "Applications" is related to the title of https://github.com/elementary/switchboard-plug-applications
-                    "settings://applications/permissions", _("System Settings"), _("Applications")
-                );
-            }
+            secondary_label_string = _("No changes were made.");
+        }
+
+        if (file is FlatpakRefFile) {
+            secondary_label_string += " ";
+            secondary_label_string += _("Visit %s for app information, updates, and to uninstall.").printf (
+                appstore_name
+            );
         }
 
+        secondary_label_string += " ";
+        secondary_label_string += _("Permissions can be changed in <a href='%s'>%s → %s…</a>").printf (
```

Comment:
Hrm this is a doozy. I'm trying to think of how to exclude the markup part from the translation. Maybe one string we want is the path:
```
var settings_path = (_("System Settings → Applications");
```
and then we want to add that into the markup:
```
var link_markup = "<a href='%s'>%s…</a>".printf ("settings://applications/permissions", settings_path);
```
and then the label string?
```
secondary_label_string += _("Permissions can be changed in %s").printf (link_markup);
```

Reply:
I mean this branch is not changing that from current master, just removing the repetition from the proposed branch. I could fix it at the same time but I'm kind of inclined to do it later since it's this way in master.

Reply:
(But yeah I do think your way makes more sense; if we want to do that here since we're touching those strings I can go ahead!)